src/components/chart/column-chart.js

```diff
@@ -15,31 +15,33 @@ class ColumnChart {
         };
     };
 
-    // Set the point padding between each bar to be 3% (an overall gap of 6%)
-    // For charts with fewer than 5 categories, we use a wider point padding of 4% (8% gap between bars)
-    // For cluster charts we use 0 for the point padding and a group padding of 4% (8% gap between bars)
-    updatePointPadding = (config, currentChart, stackedLayout, numberOfExtraLines) => {
+    // Set the point padding between each bar
+    // For charts with fewer than 5 categories, we use a wider point padding of 2%
+    // For cluster charts we use 0 for the point padding and a group padding of 2%
+    updatePointPadding = (config, currentChart, stackedLayout, numberOfExtraLines, isMobile = false) => {
         const numberOfCategories = config.xAxis.categories.length;
         const numberOfSeries = currentChart.series.length - numberOfExtraLines; // Get number of column series
+        // Guidelines
+        const categoryThreshold = isMobile ? 10 : 20;
+        const maxPointWidth = isMobile ? 55 : 75;
+
         let pointPadding = 0;
         let groupPadding = 0;
+
         // non-clustered charts or stacked charts
         if (numberOfSeries === 1 || stackedLayout === true) {
-            if (numberOfCategories > 5) {
-                pointPadding = 0.03;
-            } else {
-                pointPadding = 0.04;
-            }
+            pointPadding = numberOfCategories >= categoryThreshold ? 0.1 : 0.2;
```

Comment:
I don't think this is quite correct - we're ending up with a spacing of about 40% at desktop. If you look at my original comments, they clarify how the the pointPadding relates to the final overall gap - it's like padding in CSS - if you had 2 divs next to each other, each with a 20px left and right padding, you'd end up with an overall gap of 40px.